src/ppx/ppx_eliom_utils.ml

```diff
@@ -464,22 +464,27 @@ end
    ]
 *)
 module Rpc = struct
-  (*type error= 
+  type error=
     |No_arguments
```

Comment:
I know there is no ocamlformat on eliom but you should try to follow the formatting conventions of the rest of the file. 
(spacing around operators, bars and so on)

Reply:
Ok you cleaned it in the next commit.